### ISSUE TITLE
🐛 Server closed connection unexpectedly

### ISSUE THREAD

**Issue by kafkiansky** [cycle/database#75]
### No duplicates 🥲.

- [X] I have searched for a similar issue in our bug tracker and didn't find any solutions.

### Database

PostgreSQL

### What happened?

A bug happened!

Sometimes this error occurred:

```
General error: 7 server closed the connection unexpectedly \tThis probably means the server terminated abnormally \tbefore or while processing the request
```

Perhaps this error should be mapped as `ConnectionException` and statement must be retried.

### Version

```shell
database 14.2
PHP 8.1
```

**Comment by roxblnfk** [comment]
Probably it was fixed with #74

**Comment by roxblnfk** [comment]
covered in https://github.com/cycle/database/pull/79
